community/neo4j/src/test/java/org/neo4j/dbms/database/DefaultDatabaseManagerIT.java

```diff
@@ -24,6 +24,7 @@
 import org.junit.jupiter.api.Test;
 
 import java.util.ArrayList;
```

Comment:
This looks to be unused now.

Reply:
Thanks for looking into this.

I am afraid `ArrayList` is still used, since we say:

        List<NamedDatabaseId> databaseNames = new ArrayList<>( databases.keySet() );